airflow/ui/src/pages/Variables/Variables.tsx

```diff
@@ -64,22 +77,49 @@ export const Variables = () => {
       limit: pagination.pageSize,
       offset: pagination.pageIndex * pagination.pageSize,
       orderBy,
+      variableKeyPattern: Boolean(variableKeyPattern)
+        ? `%${variableKeyPattern}%`
```

Comment:
```suggestion
        ? `${variableKeyPattern}`
```

Wildcard is already handled in the backend. I don't think we need that in the front-end, we can remove it, same for `DagList` i think both should be removed.

Reply:
Yaa, thats right! let me rectify it.